src/Stoa.ts

```diff
@@ -145,7 +144,8 @@ class Stoa extends WebService
                 // Nothing found
                 if (!rows.length)
                 {
-                    res.status(204).send();
+                    // Service Unavailable
+                    res.status(503).send("Stoa is currently unavailable.");
```

Comment:
How about this phrase?
`No validator exists for block height.`

Reply:
It would be nice if the block height was also displayed.

Reply:
Block height is optional for parameters.
Also, the `/validators` service should always be serviceable. 
The lack of data can mean that the Stoa is preparing or the DB is empty.

Reply:
However, this may require a response to an incorrect request height.